inst/startup/worker.R

```diff
@@ -71,7 +72,24 @@ setwd(batchTaskWorkingDirectory)
 
 azbatchenv <-
   readRDS(paste0(batchJobPreparationDirectory, "/", batchJobEnvironment))
-argsList <- azbatchenv$argsList
+
+tryCatch({
+  print("Data Set Boolean: ")
+
+  print(isDataSet)
+  print(typeof(isDataSet))
+  if (isDataSet) {
```

Comment:
This really feels like something we can unit test pretty easily, hint :).

Reply:
Added a test for isDataSet